bentoml/deployment/serverless.py

```diff
@@ -92,3 +98,28 @@ def deploy_with_serverless(platform, archive_path, extra_args):
     output_path = generate_serverless_bundle(bento_service, platform, archive_path, extra_args)
     subprocess.call(['serverless', 'deploy'], cwd=output_path)
     return output_path
+
+
+def delete_serverless_deployment(platform, service_name, additional_info):
```

Comment:
`additional_info.stage` is not used?

Reply:
might be cleaner to just list out parameters than hiding them in a dict `delete_serverless_deployment(platform, service_name, region, stage):`

Reply:
done